runtime/parachains/src/scheduler.rs

```diff
@@ -113,14 +113,21 @@ pub mod pallet {
 	pub(crate) type SessionStartBlock<T: Config> = StorageValue<_, BlockNumberFor<T>, ValueQuery>;
 
 	/// One entry for each availability core. The `VecDeque` represents the assignments to be scheduled on that core.
+	/// `None` is used to signal to not schedule the next para of the core as there is one currently being scheduled.
```

Comment:
I don't understand this line. How is made the relation between `Para` and `None`?